eval_test.go

```diff
@@ -77,6 +77,16 @@ func TestEval(t *testing.T) {
 			map[string]interface{}{"a": uint(9)},
 			true,
 		},
+		testCase{
+			"a == 9",
+			map[string]interface{}{"a": "(10-1)"},
+			true,
+		},
+		testCase{
+			"a == 9",
+			map[string]interface{}{"a": "(b-1)", "b": "10"},
```

Comment:
`"10"` in quotes will be parsed as expression too, and will be numeric 10, is that intentional?

Reply:
It was intentional at the time, but kind of highlights the ambiguity that you were talking about :D This case will fail with the new commit.